packages/ui-gui/package.json

```diff
@@ -1,28 +1,35 @@
 {
   "name": "@webpack-cli/ui-gui",
   "version": "0.0.1",
+  "license": "MIT",
   "description": "GUI for webpack-cli ui",
-  "main": "index.js",
+  "main": "src/index.js",
   "scripts": {
-    "build": "webpack --config=webpack.prod.js",
-    "serve": "webpack-dev-server --config=webpack.prod.js"
+    "start": "cross-env NODE_ENV=production && npm run -s serve || npm run -s dev",
```

Comment:
If I get your question,
`&&` has higher preference over `||`
There for if `NODE_ENV=production` is true then serve will run if `NODE_ENV=production` is false then due to `&&` serve will not run instead `dev` will run.

Reply:
@dhruvdutt that's exactly what @rishabh3112, the only point here is that we're using `if-env`, since it's deprecated, we changed for `cross-env`, but `cross-env` isn't build to do this kind of `if/else`, like a said here: https://github.com/webpack/webpack-cli/pull/671#discussion_r237272276 ... Based in that, I'll update change for `per-env`, since it's proposed in `if-env` repo.